proto/vtctlservice.proto

```diff
@@ -28,3 +28,11 @@ import "vtctldata.proto";
 service Vtctl {
   rpc ExecuteVtctlCommand (vtctldata.ExecuteVtctlCommandRequest) returns (stream vtctldata.ExecuteVtctlCommandResponse) {};
 }
+
+// Service Vtctld exposes gRPC endpoints for each vt command.
+service Vtctld {
+  // GetKeyspace reads the given keyspace from the topo and returns it.
+  rpc GetKeyspace(vtctldata.GetKeyspaceRequest) returns (vtctldata.Keyspace) {};
```

Comment:
Can you fix this before we merge? Return type should be `GetKeyspareResponse`.
The rest looks good :+1:

Reply:
Done! Looking at that conflict now

Reply:
FWIW, we wrap everything in a response internally like you are suggesting, but I was surprised to see that the current Google recommendations are to return the object itself on a `Get` https://google.aip.dev/131

Reply:
oh! that's interesting..
maybe we _should_ adopt that in that case for the new RPCs (they are new after all).
@ajm188 we can merge this PR as-is but revisit the return types.

Reply:
Yeah I was kind of wondering about that ... I don't think the `Response` wrappers actually get you any additional compatibility guarantees. If I have a response like `Response: {MyObject: {Field1, Field2}}` vs `MyObject: {Field1, Field2}`, if I need to remove Field2, either way it's going to break the client, and if I want to add a field, either way it's optional to the client and goes unnoticed.

Anyway, happy to revisit this. Right now trying to figure out why after merging master and running `go mod tidy` I'm seeing:

```
❯ golangci-lint run
WARN [runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: [buildssa@vitess.io/vitess/go/test/endtoend/onlineddl [vitess.io/vitess/go/test/endtoend/onlineddl.test]: analysis skipped: errors in package: [/Users/amason/work/vitess/go/test/endtoend/onlineddl/onlineddl_test.go:209:9: undeclared name: testAlterTable]] 
ERRO Running error: bodyclose: failed prerequisites: [buildssa@vitess.io/vitess/go/test/endtoend/onlineddl [vitess.io/vitess/go/test/endtoend/onlineddl.test]: analysis skipped: errors in package: [/Users/amason/work/vitess/go/test/endtoend/onlineddl/onlineddl_test.go:209:9: undeclared name: testAlterTable]] 
ERRO Timeout exceeded: try increasing it by passing --timeout option 
```

Reply:
Ah, it looks like [this PR](https://github.com/vitessio/vitess/commit/7703097743218d2a0b64351df71947ebbdc2f673) missed one rename

Reply:
Fixed in https://github.com/vitessio/vitess/pull/7148